os/board/rtl8730e/src/component/bluetooth/example/ble_central/ble_tizenrt_client.c

```diff
@@ -344,6 +344,24 @@ trble_result_e rtw_ble_client_connect(trble_conn_info* conn_info, bool is_secure
     return TRBLE_SUCCESS;
 }
 
+trble_result_e rtw_ble_client_conn_param_update(trble_conn_handle *conn_handle, trble_conn_param *conn_param)
+{
+	rtk_bt_le_update_conn_param_t param;
+
+	param.conn_handle = *conn_handle;
+	param.conn_interval_min = conn_param->min_conn_interval;
```

Comment:
Isn't it needed NULL checking `conn_handle`?

Reply:
I have added the null check as suggested